src/wechaty_puppet_hostie/puppet.py

```diff
@@ -416,9 +416,24 @@ async def message_forward(self, to_id: str, message_id: str):
         :param message_id:
         :return:
         """
-        # TODO -> we should get the type of the message, and forward the message
-        # to different conversation_id
-        # await self.puppet_stub.message_send_mini_program()
+        payload = await self.message_payload(message_id=message_id)
+        if payload.type == MessageType.MESSAGE_TYPE_TEXT:
+            if not payload.text:
+                raise Exception('no text')
+            await self.message_send_text(conversation_id=to_id, message=payload.text)
+        elif payload.type == MessageType.MESSAGE_TYPE_URL:
+            url_link = await self.message_url(message_id=message_id)
+            await self.message_send_url(conversation_id=to_id, url=url_link)
```

Comment:
The type of the result of `message_url` is `UrlLinkPayload`, which means that the type of `url_link` is `UrlLinkPayload`.

But the type of `url` param in `message_send_url` function is `str`, so how does this pass the CI ? 😂